### ISSUE TITLE
Update Demo content to utilize Card Title fields

### ISSUE THREAD

**Issue by camikazegreen** [az-digital/az_quickstart#2197]
Our current demo content for Cards use Text fields to provide titles for each Card group. 
Once #2192 is merged we will want to update the demo content to use the new Card group title on this page: /pages/card-decks

## Conditions of satisfaction
- [ ] Text areas are no longer used for card group titles
- [ ] Each card group has an appropriate title
